cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -162,37 +137,17 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
                 "times. For example: --label distribution=external"
     ).associate()
 
-    private val scannerOption by mutuallyExclusiveOptions(
-        option(
-            "--scanner", "-s",
-            help = "The scanner to use.\nPossible values are: ${Scanner.ALL}"
-        ).convertToScanner(),
-        option(
-            "--experimental-scanners",
-            help = "A comma-separated list of experimental scanners to use. This option is mutually exclusive with " +
-                    "'--scanner'. The experimental scanner implementation scans by provenance instead of by package. " +
-                    "This improves reuse of stored scan results and increases performance if multiple packages are " +
-                    "coming from the same source code repository. The experimental scanner is work in progress and " +
-                    "it is therefore not recommended to use it in production.\n" +
-                    "Possible values are: ${ScannerWrapper.ALL}"
-        ).convertToScannerWrapperFactories()
-    ).single().default(ScannerOption.Stable(ScanCode.Factory()))
-
-    private val projectScannerOption by mutuallyExclusiveOptions(
-        option(
-            "--project-scanner",
-            help = "The scanner to use for scanning the source code of projects. By default, projects and packages " +
-                    "are scanned with the same scanner as specified by '--scanner'.\n" +
-                    "Possible values are: ${Scanner.ALL}"
-        ).convertToScanner(),
-        option(
-            "--experimental-project-scanners",
-            help = "A comma-separated list of experimental scanners to use for scanning the source code of projects. " +
-                    "By default, projects and packages are scanned with the same scanner as specified by " +
-                    "'--experimental-scanner'.\n" +
-                    "Possible values are: ${ScannerWrapper.ALL}"
-        ).convertToScannerWrapperFactories()
-    ).single()
+    private val scanners by option(
+        "--scanners", "-s",
+        help = "A comma-separated list of scanners to use.\nPossible values are: ${ScannerWrapper.ALL}"
+    ).convertToScannerWrapperFactories().required()
```

Comment:
Is this intentionally now `required` instead of providing a default scanner? IMO we should stick to ScanCode as a default scanner.

Reply:
No, this was an oversight.